src/libs/ReportActionsUtils.ts

```diff
@@ -661,30 +661,30 @@ function isNotifiableReportAction(reportAction: OnyxEntry<ReportAction>): boolea
     return actions.includes(reportAction.actionName);
 }
 
-function getChannelLogMemberAction(reportAction: OnyxEntry<ReportAction>): MessageActionItemChannelLog[] {
+function getMemberChangeMessageElements(reportAction: OnyxEntry<ReportAction>): MessageActionItemChannelLog[] {
     const messageItems: MessageActionItemChannelLog[] = [];
```

Comment:
Yeah, we'll also need to rename all "items" to "elements" across the PR. Sorry. But this will be the end of renaming things (I hope).

Reply:
@namhihi237 bump